packages/teams-js/test/public/settings.spec.ts

```diff
@@ -71,46 +57,131 @@ describe('settings', () => {
     });
   });
 
-  it('should successfully get settings', done => {
-    utils.initializeWithContext('settings').then(() => {
-      settings.getSettings(settings => {
-        expect(settings).toBe(expectedSettings);
-        done();
-      });
-
-      const message = utils.findMessageByFunc('settings.getSettings');
-      expect(message).not.toBeNull();
-
+  describe('Testing settings.getSettings function', () => {
+    const allowedContexts = [
+      FrameContexts.content,
+      FrameContexts.settings,
+      FrameContexts.remove,
+      FrameContexts.sidePanel,
+    ];
+    const expectedSettings: settings.Settings = {
+      suggestedDisplayName: 'someSuggestedDisplayName',
+      contentUrl: 'someContentUrl',
+      websiteUrl: 'someWebsiteUrl',
+      entityId: 'someEntityId',
+    };
+
+    it('settings.getSettings should not allow calls before initialization', () => {
       const expectedSettings: settings.Settings = {
```

Comment:
This is also defined on line 67-72. Should it be removed from within the test case?

Reply:
Removed duplicates.